components/whyd/2022/Terminal/UserInputArea.vue

```diff
@@ -284,4 +296,12 @@ export default {
   opacity: 0;
   position: absolute;
 }
+
+.red-text {
+  color: red;
```

Comment:
Nit: can we make this a more maroon-like color? This is partially my personal preference and partially that I think it will reduce jitter